wgpu-types/src/instance.rs

```diff
@@ -216,6 +216,8 @@ impl BackendOptions {
 pub struct GlBackendOptions {
     /// Which OpenGL ES 3 minor version to request, if using OpenGL ES.
     pub gles_minor_version: Gles3MinorVersion,
+    /// TODO DOCS
```

Comment:
plz do so before merging!
Although I figure that will mostly just point to the `GlFenceBehavior` doc